umaserverdemo/src/main/kotlin/com/lightspark/Vasp1RequestCache.kt

```diff
@@ -34,7 +34,13 @@ class Vasp1RequestCache {
 
     fun saveLnurlpResponseData(lnurlpResponse: LnurlpResponse, receiverId: String, vasp2Domain: String): String {
         val uuid = UUID.randomUUID().toString()
-        lnurlpRequestCache[uuid] = Vasp1InitialRequestData(lnurlpResponse, receiverId, vasp2Domain)
+        lnurlpRequestCache[uuid] = Vasp1InitialRequestData(lnurlpResponse, nonUmaLnurlpResponse = null, receiverId, vasp2Domain)
+        return uuid
+    }
+
+    fun saveNonUmaLnurlpResponseData(nonUmaLnurlpResponse: NonUmaLnurlpResponse, receiverId: String, vasp2Domain: String): String {
```

Comment:
I think we might not need this due to the LNURL fallback stuff now, but let me know if it's still used. I was able to remove it from other vasps.

Reply:
hm I may be misunderstanding - if we remove this, would you be able to send regular LNURL to an uma v0 vasp?

Reply:
Yeah you should still be able to. NonUma and UMA LnurlpResponse objects are just the same object definition now, so we don't need a separate function or storage mechanism to save them